brion/plugin/compartmentReportBinary.cpp

```diff
@@ -653,6 +663,10 @@ bool CompartmentReportBinary::_parseMapping()
 
         if (_header.byteswap)
```

Comment:
just out of curiosity: is there a way to combine get<> with byteswap if needed?
Could help to avoid funny errors if new get<>s are added for some reason.